app/src/app/navtitle/navtitle.component.html

```diff
@@ -3,33 +3,61 @@ <h1 i18n><a routerLink="/">Itero</a></h1>
   <h2 i18n>Online iterative vote <span class="long-title">application</span></h2>
   <nav>
     <a *ngIf="(demoIsActive$ | async) && !(session.state$ | async).logged"
-      [routerLink]="(demoPath$ | async)" routerLinkActive="active">
+      [routerLink]="(demoPath$ | async)" routerLinkActive="active" class="nav-link">
       <span class="navitem" i18n>Try it</span>
     </a>
-    <a *ngIf="(session.state$ | async).logged" routerLink="/r/list" routerLinkActive="active">
+    <a *ngIf="(session.state$ | async).logged" routerLink="/r/list" routerLinkActive="active" class="nav-link">
       <span class="navitem" i18n>Polls</span>
     </a>
-    <a *ngIf="!(session.state$ | async).logged" routerLink="/r/session/signup" routerLinkActive="active">
+    <a *ngIf="!(session.state$ | async).logged" routerLink="/r/session/signup" routerLinkActive="active" class="nav-link">
       <span class="navitem" i18n>Sign up</span>
     </a>
-    <a *ngIf="!(session.state$ | async).logged" routerLink="/r/session/login" routerLinkActive="active">
+    <a *ngIf="!(session.state$ | async).logged" routerLink="/r/session/login" routerLinkActive="active" class="nav-link">
       <span class="navitem" i18n>Log in</span>
     </a>
-    <a *ngIf="(session.state$ | async).logged"  routerLink="/r/create" routerLinkActive="active">
+    <a *ngIf="(session.state$ | async).logged" routerLink="/r/create" routerLinkActive="active" class="nav-link">
       <span class="navitem">Create</span>
     </a>
-    <a *ngIf="(session.state$ | async).logged" [matMenuTriggerFor]="profileMenu">
+    <a [matMenuTriggerFor]="profileMenu" id="naviconmenu">
+      <span class="navitem"><mat-icon>menu</mat-icon></span>
+    </a>
+    <a *ngIf="(session.state$ | async).logged" [matMenuTriggerFor]="profileMenu" id="naviconperson">
       <span class="navitem"><mat-icon>person</mat-icon></span>
     </a>
   </nav>
 </div>
 <mat-menu #profileMenu="matMenu" class="nav-menu" xPosition="before">
-  <button mat-menu-item (click)="notify.askAUthorization()" [disabled]="(notify.permission$ | async) != 'default'">
-    <ng-container [ngSwitch]="notify.permission$ | async">
-      <mat-icon *ngSwitchCase="'default'">notifications</mat-icon>
-      <mat-icon *ngSwitchCase="'denied'">notifications_off</mat-icon>
-      <mat-icon *ngSwitchCase="'granted'">notifications_active</mat-icon>
-    </ng-container> Notifications
-  </button>
-  <button mat-menu-item (click)="logoff()"><mat-icon>logout</mat-icon> Log off</button>
+  <div id="menubutton">
+    <button *ngIf="(demoIsActive$ | async) && !(session.state$ | async).logged" mat-menu-item class="nav-link" [routerLink]="(demoPath$ | async)" routerLinkActive="active">
```

Comment:
This line is much too long (172 characters). I try to respect a soft limit of 100 and a hard limit of 120. Considering this hard limit, lines 12, 15, 35, 39, 43 and 47 are too long too.
```suggestion
    <button *ngIf="(demoIsActive$ | async) && !(session.state$ | async).logged" mat-menu-item class="nav-link" 
            [routerLink]="(demoPath$ | async)" routerLinkActive="active">
```